src/dependency-manager/src/manager.ts

```diff
@@ -29,7 +29,7 @@ export default abstract class DependencyManager {
         local_path: component.getLocalPath(),
         artifact_image: component.getArtifactImage(),
       });
-      node.instance_id = `${Refs.safeRef(component.getRef())}-local`;
+      node.instance_id = `${Refs.safeRef(component.getRef())}-component`;
```

Comment:
Missed this in the initial review. Remove setting node.instance_id here. Set in the local-manager getGraph method when we loop over the nodes for a component. 

https://github.com/architect-team/architect-cli/blob/master/src/common/dependency-manager/local-manager.ts#L45